### ISSUE TITLE
Spigot 1.16.1 support

### ISSUE THREAD

**Issue by Srborjaa** [AcmeProject/WildernessTp#81]
Hi, I know that this is not the best place to post that, but could you update Wilderness-TP to 1.16?

Thanks so much :P

**Comment by RoboMWM** [comment]
what's wrong with it?

**Comment by Srborjaa** [comment]
Yesterday I saw the plugin page but I didn't see if it has official support for the 1.16 version, so I decided to post it to avoid compatibility errors.

**Comment by RoboMWM** [comment]
I have several plugin pages and I can't update all the checkboxes nor have the time to test all of them, but because of the way the Bukkit API is designed, most plugins should be forward-compatible. Testing would be appreciated.

**Comment by GrilledCow** [comment]
Running Paper 21 on 1.16, plugin version is 219-2db6004 and I believe this is an error related to the plugin: https://hastebin.com/acadajareg.bash seems to fire whenever someone issues /rtp. Hopefully this is helpful, everything still seems to work.

**Comment by RoboMWM** [comment]
Strange error, not sure if that's the plugin's fault - seems like an error being thrown by the server.

**Comment by GrilledCow** [comment]
> Strange error, not sure if that's the plugin's fault - seems like an error being thrown by the server.

Yeah I've looked around and it's not the plugin, it happens on most plugins that teleport players, I just assumed whenever I saw it happen after someone /rtp'd it would've been issued with the plugin. It has been reported: https://bugs.mojang.com/browse/MC-191388
